lib/isaac/isaac_defines.hpp

```diff
@@ -54,7 +54,7 @@
 #endif
 
 #define ISAAC_MAX_RECEIVE 262144 //256kb
-#define ISAAC_Z_NEAR 1.0f
```

Comment:
why do you change this value? Could you give me an explanation of what this variable is doing?

Reply:
This value defines the distance to the camera frustums near plane. It is usually a small float value and in that specific instance it defines the starting position of the raycasting ray.
For the particle rendering was a value of 1.0 too far away and particles which were near the camera were cut on the near plane.
In the volume rendering were no artifacts, because it isn't visually obvious where the ray starts.

![canvas_clipping](https://user-images.githubusercontent.com/22729279/58869198-0eb28700-86be-11e9-9d6d-4e5449df5973.png)